tokio/src/task/blocking.rs

```diff
@@ -1,7 +1,7 @@
 use crate::task::JoinHandle;
 
 cfg_rt_threaded! {
-    /// Runs the provided blocking function without blocking the executor.
+    /// Runs the provided closure on a thread where blocking is acceptable.
```

Comment:
The `block_in_place` function works differently from `spawn_blocking`. It doesn't run it on a separate thread.

Reply:
Sorry about that. I am making the change. I modified the wrong function by mistake.